recipes/ray-packages/meta.yaml

```diff
@@ -0,0 +1,256 @@
+        # gpustat-0.6.0 has a dependency which does not exist on Windows;
+        # skip it there until gpustat is fixed as it is optional
+        - gpustat  # [not win]
+        - grpcio >=1.28.1
+        - jsonschema
+        - msgpack-python >=1.0.0, <2.0.0
+        - numpy >=1.16
+        - protobuf >=3.8.0
+        - psutil
+        - pyyaml
+        - requests
+        - redis-py >=3.3.2, <3.5.0
+        - opencensus
+        - prometheus_client >=0.7.1
+        - pickle5  # [py<38]
+        - funcsigs
+        - setproctitle
+    test:
+      imports:
+        - ray
+        - ray._raylet
+        - ray.actor
+        - ray.profiling
+        - ray.runtime_context
+        - ray.state
+        - ray.worker
+
+  - name: ray-debug
+    build:
+      # some weird incompatibility when installing during tests... disabling for now
+      skip: True  # [not win]
+      # there is no py-spy on Windows
+      skip: True  # [win]
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - py-spy >=0.2.0
+    test:
+      imports:
+        # there doesn't appear to be a debug specific module
+        - ray
+
+  - name: ray-dashboard
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+    test:
+      imports:
+        - ray.dashboard
+
+  - name: ray-rllib
+    build:
+      # needed atari deps are missing on Windows
+      skip: True  # [win]
+      entry_points:
+        - rllib = ray.rllib.scripts:cli
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - tabulate
+        - tensorboardX
+        - pandas
+        - dataclasses  # [py < 37]
+        - atari_py
+        - dm-tree
+        - gym-atari
+        - lz4
+        # until there is a build of opencv-python-headless which does not require
+        # full graphics stack we have to rely on user system having needed
+        #  graphics libraries installed
+        - opencv <=4.30.0
+        - pyyaml
+        - scipy
+    test:
+      imports:
+        - ray.rllib
+
+  - name: ray-serve
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - uvicorn
+        - flask
+        - requests
+        - pydantic <1.7
+        - dataclasses  # [py < 37]
+    test:
+      imports:
+        - ray.serve
+
+  - name: ray-tune
+    build:
+      entry_points:
+        - tune = ray.tune.scripts:cli
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - tabulate
+        - tensorboardX
+        - pandas
+        - dataclasses  # [py < 37]
+    test:
+      imports:
+        - ray.tune
+
+  - name: ray-autoscaler
+    build:
+      skip: True  # [win]
```

Comment:
Could we add a comment here which deps are missing on windows?

Reply:
It has nothing to do with deps, it's autoscaler itself not working on Windows.
It relies on having POSIX-compliant `ssh` and `rsync` binaries.

If you enable native `ssh` on Windows it _seems_ to start working, but eventually fails somewhere in the internals (note that I have last checked that with Ray 0.8.7, but I highly doubt things changed much in 1.0).

Reference: https://github.com/ray-project/ray/issues/9114